mc_automation_tools/base_requests.py

```diff
@@ -156,11 +163,28 @@ def send_delete_request(url, params=None):
     try:
```

Comment:
Where is the exception ? Except requests.RequestException as e: 
like this ? 
 except requests.RequestException as e:
        _log.error("Request failed: %s", str(e))
        raise